source/jormungandr/jormungandr/scenarios/ridesharing/instant_system.py

```diff
@@ -37,12 +37,15 @@
 from jormungandr import utils
 from jormungandr import app
 import jormungandr.scenarios.ridesharing.ridesharing_journey as rsj
-from jormungandr.scenarios.ridesharing.ridesharing_service import AbstractRidesharingService
+from jormungandr.scenarios.ridesharing.ridesharing_service import AbstractRidesharingService, Rs_FeedPublisher
+
+DEFAULT_INSTANT_SYSTEM_FEED_PUBLISHER = None
```

Comment:
I'd still put something like private, maybe https://instant-system.com/disclaimers/disclaimer_XX.html (it's not a real url, but it's the idea)